sim/src/main/makefrag/firesim/Makefrag

```diff
@@ -34,6 +34,10 @@ common_chisel_args = $(patsubst $(base_dir)/%,%,$(GENERATED_DIR)) $(DESIGN_PACKA
 
 ifdef FIRESIM_STANDALONE
 	firesim_sbt_project := {file:${firesim_base_dir}/target-rtl/chipyard/}firechip
+
+	lookup_scala_srcs = $(shell find -L $(1)/ -name target -prune -o -iname "*.scala" -print 2> /dev/null)
```

Comment:
So there's no way to reuse this from Chipyard?

Reply:
Not easily. I guess we could make a separate makefrag in Chipyard that we can source without worrying about polluting environment variables.